### ISSUE TITLE
'ProgressTracker' object has no attribute '_learning_rate'

### ISSUE THREAD

**Issue by tboneai** [uber/ludwig#742]
**Describe the bug**
Upgraded Ludwig (pip install) this weekend and now I am receiving an error when using learning rates. The file runs fine until it gets to the point that its called. When I remove the 2 lines from the mdf file it runs fine. 

**To Reproduce**
Ran experiment command

Please provide code, yaml definition file and a sample of data in order to entirely reproduce the issue. Issues that are not reproducible will be ignored.

[mdf_test_num_16v5_git.txt](https://github.com/uber/ludwig/files/4783258/mdf_test_num_16v5_git.txt)

[EURUSD_sample.txt](https://github.com/uber/ludwig/files/4783271/EURUSD_sample.txt)

**Expected behavior**
Expected learning rate to change and experiment to continue running. 

**Screenshots**
[New Text Document (2).txt](https://github.com/uber/ludwig/files/4783245/New.Text.Document.2.txt)

**Environment (please complete the following information):**
 - OS: [e.g. iOS] Windows 10 vers 10.0.18363 build 18363
- Python version 3.6 
- Ludwig version 0.2.2.7

**Additional context**

**Comment by w4nderlust** [comment]
Pushed a commit that should have fixed the issue.
Cna you please install from master with `pip uninstall ludwig & pip install git+http://github.com/uber/ludwig.git` and confirm?
